packages/preview/lyceum/0.1.0/stencil/main.typ

```diff
@@ -0,0 +1,136 @@
+// #import "@preview/lyceum:0.1.0": FRONT-MATTER
+#import "@preview/lyceum:0.1.0": FRONT-MATTER, BODY-MATTER, APPENDIX, BACK-MATTER
+
+#let TEXT-SIZE = 11pt
+
+//----------------------------------------------------------------------------//
+//                                FRONT-MATTER                                //
+//----------------------------------------------------------------------------//
+
+#show: FRONT-MATTER.with(
+  // Document metadata
+  title: (
+    title: "Igneous Rocks",
+    subtitle: "The Hard Science",
+    sep: " - "
+  ),
+  authors: (
+    (
+      given-name: "Evelyn D.",
+      name: "Crump",
+      affiliation: "Rocks Hard Research Group",
+      email: "crumped@rockshard.org.far",
+      location: "Rich Mines, Faraway Country",
+    ), (
+      preffix: "Sir.",
+      given-name: "Effie J.",
+      name: "Hitchcock",
+      suffix: "Jr.",
+      affiliation: "Hard University",
+      email: "hitchcockej@hard.edu.far",
+      location: "Rockbridge, Faraway Country",
+    ),
+  ),
+  editors: ("Cenhelm, Erwin", ),
+  publisher: "Lyceum Publisher",
+  location: "Lyceum City, Faraway Country",
+  affiliated: (
+    illustrator: ("Revaz Sopheap", ),
+    organizer: "Darko Sergej",
+  ),
+  keywords: ("igneous", "rocks", "geology", ),
+  date: datetime(year: 2024, month: 9, day: 13), // auto => datetime.today()
+  // Document general format
+  page-size: (width: 155mm, height: 230mm),
+  page-margin: (inside: 30mm, rest: 25mm),
+  page-binding: left,
+  page-fill: color.hsl(45deg, 15%, 85%),  // ivory
+  text-font: ("EB Garamond", "Linux Libertine"),
```

Comment:
Linux Libertine is no longer the default font in Typst because the Libertinus Serif fork is better maintained, maybe you want to change that here too?

```suggestion
  text-font: ("EB Garamond", "Libertinus Serif"),
```

Reply:
Surely I'll change there too. Sorry to miss that one instance / Thanks for the diagnostic and suggestion, which I accept.

I've never had this sort of (pull-request's commit amidst comments) before... not sure how's the best way to proceed to accept it, but since I want such a commit on my side (as to tag it properly), I'll proceed by re-committing from my side, which has been automatically updating the PR. This time, it looks like it will go through without errors/warnings.

Reply:
Thank you, @elegaanz !